internal/backend/db/dbgorm/connections.go

```diff
@@ -21,12 +22,30 @@ func (gdb *gormdb) createConnection(ctx context.Context, conn *scheme.Connection
 	return gdb.createEntityWithOwnership(ctx, createFunc, conn, conn.ProjectID)
 }
 
+func (gdb *gormdb) deleteConnectionsAndVars(where string, id sdktypes.UUID) error {
+	// should be transactional with context already applied
+
+	var ids []sdktypes.UUID
+	q := gdb.db.Model(&scheme.Connection{})
+	q = q.Clauses(clause.Returning{Columns: []clause.Column{{Name: "connection_id"}}})
```

Comment:
try something like this please where = fmt.Sprintf("%x = ?", where)

Reply:
done